c++/src/Statistics.cc

```diff
@@ -317,6 +317,8 @@ namespace orc {
       _stats.setMaximum(0);
       _lowerBound = 0;
       _upperBound = 0;
+      _minimumNano = DEFAULT_MIN_NANOS;
```

Comment:
This looks inconsistent. We use `NANOS` in `DEFAULT_MIN_NANOS` and `Nano` in `_minimumNano`.
Maybe, `_minimumNano` -> `_minimumNanos`?

Reply:
We already use `nanos` in `stats.has_minimumnanos()`, too.